fastlane/lib/fastlane/actions/unlock_keychain.rb

```diff
@@ -72,7 +72,8 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :path,
                                        env_name: "FL_UNLOCK_KEYCHAIN_PATH",
                                        description: "Path to the keychain file",
-                                       optional: false),
+                                       default_value: "login.keychain",
+                                       optional: true),
```

Comment:
Can we change this to `optional: false`? We can still have a `default_value` but we don't want this to ever be `nil` 😊

Reply:
Fixed it 🙂